resources/var/www/html/errors/starting.html

```diff
@@ -1,29 +1,31 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width">
-	<meta http-equiv="refresh" content="10" />
-	<title>Dogu is starting</title>
-	<!--Stylesheet CES Theme-->
-	<link rel="stylesheet" href="/errors/css/ces.css" media="screen">
-	<link rel="stylesheet" href="/errors/css/dogu-starting-styles.css">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta http-equiv="refresh" content="10" />
+    <title>Dogu startet</title>
+    <link rel="stylesheet" href="/errors/css/dogu-starting.css">
 </head>
 <body>
-<div class="logo">
-	<img src="/errors/img/logo-white-320px.png">
-</div>
-<div class="container">
-	<div class="row">
-		<div class="col-xs-6">
-			<div class="imageContainer flyIn">
-				<img class="flyingCloud " src="/errors/img/flying_cloud.svg">
-			</div>
-		</div>
-		<div class="col-xs-8 col-sm-6 starting">
-			<h1 id="doguName">Dogu is starting</h1>
-		</div>
-	</div>
+<div class="dogu-starting">
+    <div>
+        <img alt="" class="logo">
+    </div>
+    <div>
+        <h1>Dogu startet</h1>
+        <h2>Bitte haben Sie Geduld.</h2>
+        <div class="actions">
+            <button onClick="window.location.reload();">
+                Dogu neu laden
+                <svg width="16" height="16" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Like in cloudogu/nginx#103: I have no clue what SVG this depicts unless I saved this as a file and looked at it. Can we add a semantic way of indicating this is a reload arrow?